_topics/en/anonymity-networks.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Anonymity networks
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Tor
+  - I2P
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Bitcoin Core #21594 updates the `getnodeaddresses` RPC to Tor and I2P"
+    url: /en/newsletters/2021/04/14/#bitcoin-core-21594
+
+  - title: "Bitcoin Core #20197 updates eviction logic to keep long-running onion peers"
+    url: /en/newsletters/2021/04/07/#bitcoin-core-20197
+
+  - title: "Question: how do use I2P with Bitcoin Core?"
+    url: /en/newsletters/2021/03/31/#how-can-i-use-bitcoin-core-with-the-anonymous-network-protocol-i2p
+
+  - title: "Bitcoin  Core #20685 adds support for the I2P privacy network"
+    url: /en/newsletters/2021/03/10/#bitcoin-core-20685
+
+  - title: "Bitcoin Core GUI #162 more network type information to the GUI"
+    url: /en/newsletters/2021/01/06/#bitcoin-core-gui-162
+
+  - title: "Bitcoin Core #19954 completes the BIP155 addr v2 implementation"
+    url: /en/newsletters/2020/10/14/#bitcoin-core-19954
+
+  - title:  Bitcoin Core PR Review Club on a proposed implementation of BIP155 addr v2
+    url: /en/newsletters/2020/08/12/#bitcoin-core-pr-review-club
+
+  - title: BIP for new address relay message to support Tor v3 onion addresses
+    url: /en/newsletters/2019/03/12/#version-2-addr-message-proposed
+
+  - title: Bitcoin Core 0.21.0 released with support for Tor v3 onion addresses
+    url: /en/newsletters/2021/01/20/#bitcoin-core-0-21-0
+
+  - title: "Year-in-review: support for Tor v3 onion addresses"
+    url: /en/newsletters/2020/12/23/#addrv2
+
+  - title: "Bitcoin Core #19991 allows independently tracking Tor connections"
```

Comment:
It enables bitcoind to know when an inbound peer is a onion one. Before this PR, we used localhost as a heuristic, e.g. in the inbound eviction protection logic.

Reply:
(Outbound onion peers were always easy to detect thanks to the ".onion" in their address.)

Reply:
Yeah, I know.  We only have 75 characters to explain it and "Bitcoin Core \#19991 " takes up 20 of those characters, so if you have a better description in 55 characters, I'd love to hear it!  :-)

Reply:
Ok. Maybe "Bitcoin Core #19991 enables knowing when inbound connections are onion peers" (76 chars)